.gitignore

```diff
@@ -9,3 +9,4 @@ composer.phar
 phpunit.xml
 phpstan.neon
 /.cache/
+docs/.vscode
```

Comment:
This looks like a configuration that should be [global to your system](https://www.mashupgarage.com/playbook/git/gitignore.html) instead of specific to every project you work on.

Reply:
Thanks for the link! I'll remove this line from the .gitignore.